src/containers/home/home.component.jsx

```diff
@@ -1,49 +1,44 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 
 import { auth } from "database";
 
 import { fetchPosts } from "database/posts";
```

Comment:
Do we still need this function imported?

Reply:
resolved in second PR